subgraph-js/tsconfig.json

```diff
@@ -6,5 +6,7 @@
   },
   "include": ["src/**/*"],
   "exclude": ["**/__tests__"],
-  "references": []
+  "references": [
+    { "path": "../gateway-js" },
```

Comment:
We would also need to add gateway to subgraph's package.json dependencies, but see comment above (I hope we can avoid that)